SpringAwwCrud/build/resources/main/templates/home.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="bootstrap.min.css">
+    <title>Welcome to my C.R.U.D. Program</title>
+</head>
+    <body>
+    <div class="container">
+    {{^userName}}
```

Comment:
I would probably use `user` as something to key off of just in case the session has a username that isn't in the database.  Sessions can be manipulated.